### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by nw0428**
This cleans up the repo and puts things into their appropriate folders. @all please review

### CODE REVIEW

**Comment by nw0428** [comment]
LGTM

File: index.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
 <h1>Summer 2016 SIB Class!!!</h1>
   <div class="center">
     <ul>
-    	<li><a href="david.html">David Zabner</a></li>
+    	<li><a href="pages/david.html">David Zabner</a></li>
```
Comment (nw0428) [inline index.html:10]: I see what you did there.

**Comment by dterana** [comment]
LGTM

**Comment by Tonloc88** [comment]
LGTM2

**Comment by chdastolfo** [comment]
I c wut u did thar.

**Comment by Tonloc88** [comment]
silla amrican!

**Comment by Kira-Pilot** [comment]
Noice.

**Comment by tracyisman** [comment]
I found it! yay!

**Comment by dmasters19** [comment]
way 2 go

**Comment by oterogalvis** [comment]
Vi lo que hiciste.

**Comment by neugierige** [comment]
yo

**Comment by oterogalvis** [comment]
Esto no es un chat.

**Comment by Tonloc88** [comment]
esto no bien

**Pull request merged**